index.html

```diff
@@ -295,5 +295,20 @@ <h3 class="languages-title">Languages</h3>
         </div>
       </div>
     </section>
+    <section class="contact-section">
+      <h3>Contact me</h3>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mwkypgbn" method="post">
+        <input type="text" name="full-name" required  maxlength="30" placeholder="Full name" class="input-field-contact"><br>
+        <input type="email" name="email" required placeholder="Enter email" class="input-field-contact"><br>
+        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Write your message here" required></textarea><br>
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements: Please ensure that the message field accepts up to 500 chars. You can do this by adding the `maxlength="500"` attribute